docs/element-definitions.md

```diff
@@ -1431,7 +1431,7 @@ Example of an HTTP Transaction.
 
 - `element` - `"httpTransaction"`
 - `attributes`
-    - `authSchemes` ([Array][]) - An array of authentication and authorization schemes that apply to the transaction
+    - `authSchemes` ([Array][]\[[Auth Scheme][], [Auth Scheme Requirement][]]) - An array of authentication and authorization schemes out of which at least one SHOULD apply to the transaction.
```

Comment:
* `An array of authentication and authorization schemes of which one SHOULD match the transaction`?
* Also, should this SHOULD be a SHOULD or a MUST? :)

Reply:
`match the transaction`? Shouldn't it be `apply to the transaction`?